flink-runtime/src/main/java/org/apache/flink/runtime/util/BashJavaUtils.java

```diff
@@ -63,11 +69,39 @@ private static void getTmResourceParams(String[] args) throws Exception {
 	}
 
 	private static Configuration getConfigurationForStandaloneTaskManagers(String[] args) throws Exception {
-		Configuration configuration = TaskManagerRunner.loadConfiguration(args);
+		Configuration configuration = loadConfiguration(args);
 		return TaskExecutorProcessUtils.getConfigurationMapLegacyTaskManagerHeapSizeToConfigOption(
 			configuration, TaskManagerOptions.TOTAL_FLINK_MEMORY);
 	}
 
+	@VisibleForTesting
+	static Configuration loadConfiguration(String[] args) throws FlinkException {
```

Comment:
Some minor suggestion for future maintenance:
We could move `BashJavaUtils` to `org.apache.flink.runtime.util.bash` package and add there `FlinkConfigLoader` for `loadConfiguration`. The `BashJavaUtilsTest` would be `FlinkConfigLoaderTest`. Also options could be there:
`private final static Options CMD_OPTIONS = (new ClusterConfigurationParserFactory()).getOptions();`.